nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/IdentifyMimeType.java

```diff
@@ -136,8 +151,11 @@ public class IdentifyMimeType extends AbstractProcessor {
     private List<PropertyDescriptor> properties;
 
     private final TikaConfig config;
-    private Detector detector;
-    private MimeTypes mimeTypes;
+    private volatile Detector defaultDetector;
+    private volatile MimeTypes defaultMimeTypes;
+
+    private volatile Detector customDetector;
+    private volatile MimeTypes customMimeTypes;
```

Comment:
Is there a reason for marking these as `volatile`? They should not be changed after Processor initialization.

Reply:
volatile removed.  Since we are using only one MimeTypes, using original variable names